src/wallet/rpcwallet.cpp

```diff
@@ -2364,6 +2365,56 @@ UniValue settxfee(const JSONRPCRequest& request)
     return true;
 }
 
+UniValue setprivatesendrounds(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() != 1)
+        throw std::runtime_error(
+            "setprivatesendrounds rounds\n"
+            "\nSet the number of rounds for PrivateSend mixing.\n"
+            "\nArguments:\n"
+            "1. rounds         (numeric, required) The default number of rounds is " + std::to_string(DEFAULT_PRIVATESEND_ROUNDS) + 
+            " Cannot be more than " + std::to_string(MAX_PRIVATESEND_ROUNDS) + " nor less than " + std::to_string(MIN_PRIVATESEND_ROUNDS) +
+            "\nExamples:\n"
+            + HelpExampleCli("setprivatesendrounds", "4")
+            + HelpExampleRpc("setprivatesendrounds", "16")
+        );
+
+    
+    int nRounds = request.params[0].get_int();
+
+    privateSendClient.nPrivateSendRounds = nRounds;
```

Comment:
Why not squashing 2 lines into 1 i.e. `privateSendClient.nPrivateSendRounds = request.params[0].get_int()`? (same for `amount`)

Reply:
Ah, wait a minute... you must also make sure that values actually fin into min/max.

Reply:
Would that be best, or would it be better to return an error / return `false` if it doesn't fit in the range?

Reply:
Hmm.. good idea 👍 But instead of returning `false` should probably throw an error i.e. smth like
```
if (nRounds < MIN_PRIVATESEND_ROUNDS || nRounds > MAX_PRIVATESEND_ROUNDS)
    throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid amount");
```